### PR TITLE
Add Mexican Spanish language to the installer

### PR DESCRIPTION
**Pull request by LouiseBonnard**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Add Mexican Spanish language to the installer
| Type?         | improvement
| Category?     | IN
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Install PrestaShop in ES-MX

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10111)

### CODE REVIEW

**Comment by LouiseBonnard** [comment]
I'm not certain about this one since it is a dependant language... perhaps it is useless to add it in the installer. Let me know. :-)

File: install-dev/langs/mx/language.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<language>
+	<name><![CDATA[Español Mexicano (Mexican Spanish)]]></name>
```
Comment (LouiseBonnard) [inline install-dev/langs/mx/language.xml]: Can I add a commit to correct the line? It should be _Español MX (Spanish)_ if I refer to https://github.com/PrestaShop/PrestaShop/blob/develop/app/Resources/all_languages.json.

Comment (PierreRambaud) [reply install-dev/langs/mx/language.xml]: Of course, you can improve your Pull Request until it's merged :)

File: install-dev/langs/mx/language.xml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<language>
+	<name><![CDATA[Español MX (Spanish)]]></name>
+	<locale>es-MX</locale>
+	<language_code>es</language_code>
```
Comment (LouiseBonnard) [inline install-dev/langs/mx/language.xml:5]: Should it be _es_ or _es_mx_? cf. https://github.com/PrestaShop/PrestaShop/blob/develop/app/Resources/all_languages.json

Comment (PierreRambaud) [reply install-dev/langs/mx/language.xml:5]: No, es-MX is fine :)

**Comment by PierreRambaud** [comment]
Will be merged in #11058 Thanks @LouiseBonnard

**Pull request closed without merging**